### PR TITLE
Created a stacked style for metalists

### PR DESCRIPTION
**Pull request by ztolley**
capture

### CODE REVIEW

File: assets/stylesheets/components/_meta-list.scss [hunk 1]
```diff
@@ -72,3 +72,15 @@
     margin-right: 0;
   }
 }
+
+.c-meta-list--stacked {
+  display: block;
+
+  > .c-meta-list__item {
```
Comment (teneightfive) [inline assets/stylesheets/components/_meta-list.scss]: Do you need the immediate child selector here? Isn't it being nested within the parent class selector enough?

File: src/apps/components/views/entity-list.njk [hunk 2]
```diff
@@ -87,4 +87,9 @@
     </div>
   {% endcall %}
 
+    {% call Example('Audit log') %}
```
Comment (teneightfive) [inline src/apps/components/views/entity-list.njk]: Needs outdenting by 2 spaces.

File: assets/stylesheets/components/_meta-list.scss (hunk 1 above)
Comment (ztolley) [reply assets/stylesheets/components/_meta-list.scss]: probably. Had it there initially to stop the style affecting sub-components but then that's not going to happen

File: src/apps/components/controllers.js [hunk 3]
```diff
@@ -51,10 +51,31 @@ async function renderEntityList (req, res) {
       })
     })
 
+  const auditLog = {
+    items: [{
+      type: 'audit',
+      name: '20th December 2016 09:00am',
```
Comment (teneightfive) [inline src/apps/components/controllers.js]: This is fine because its an example but we should make sure we [write dates](https://www.gov.uk/service-manual/design/dates#how-to-write-dates) as 20 December 2016 for consistency.

Comment (teneightfive) [reply src/apps/components/controllers.js]: Also, it looks like the time is using a combination of 24 hour and am/pm.

**Comment by tyom** [comment]
Is it worth taking the opportunity to document all other variants of `MetaList`?

**Comment by ztolley** [comment]
@tyom I'd humbly say that's beyond the scope of this PR and I don't want it to hold up the audit card

File: assets/stylesheets/components/_meta-list.scss (hunk 1 above)
Comment (teneightfive) [reply assets/stylesheets/components/_meta-list.scss]: Correct. One of the points of using BEM.

File: src/apps/components/controllers.js [hunk 4]
```diff
@@ -51,10 +51,31 @@ async function renderEntityList (req, res) {
       })
     })
 
+  const auditLog = {
+    items: [{
+      type: 'audit',
+      name: '20 December 2016 09:00am',
```
Comment (teneightfive) [inline src/apps/components/controllers.js]: `9:00am`?

File: src/apps/components/views/entity-list.njk [hunk 5]
```diff
@@ -21,7 +21,7 @@
           {% for company in companiesSearch.companies %}
             <li class="c-entity-list__item">
               <h3 class="c-entity-list__item-header">
-                <a href="/viewcompanyresult/{{ company.id }}">{{ company.name}}</a>
+              <a href="/viewcompanyresult/{{ company.id }}">{{ company.name}}</a>
```
Comment (teneightfive) [inline src/apps/components/views/entity-list.njk]: Unnecessary change?

File: src/apps/components/controllers.js (hunk 4 above)
Comment (ztolley) [reply src/apps/components/controllers.js]: I have missed your attention to detail :)

**Comment by jim68000** [comment]
🤘

File: src/apps/components/controllers.js (hunk 4 above)
Comment (teneightfive) [reply src/apps/components/controllers.js]: It's all about the polish :)
